pkg/solana/genesis/assets/primordial.yaml.tmpl

```diff
@@ -0,0 +1,7 @@
+{{- range .Primordial }}
+{{ .Pubkey }}:
+  balance: {{ .Lamports }}
+  owner: 11111111111111111111111111111111
+  executable: false
+  data:
+{{ end -}}
```

Comment:
This is super nice.

An FYI that we always set data to empty right now but we could seed default for programs other than system which store data like tokens or custom programs.

We would need serializers to map the objects to their byte layout. This info is available in the program IDL.